frontend/app/components/wp-edit/wp-edit-field/wp-edit-duration-field.directive.html

```diff
@@ -0,0 +1,5 @@
+<input type="number"
+       wp-edit-field-requirements="vm.field.schema"
+       ng-model="vm.workPackage[vm.fieldName]"
+       duration-value
+       ng-blur="vm.deactivate()">
```

Comment:
You should add the `focus` directive to let the field be focused after triggering inline edit.